src/content/structured/components/tooltips/code.mdx

```diff
@@ -19,20 +19,28 @@ tabs:
   ]
 ---
 
-import { IcTooltip, IcButton } from "@ukic/react";
+import { IcTooltip, IcButton, IcChip } from "@ukic/react";
 
 ## Component demo
 
 export const snippets = [
   {
     language: "Web component",
-    snippet: `<ic-tooltip target="text-button" label="This is a description of the button" placement="top"></ic-tooltip>
-<ic-tooltip target="text-button" label="This is a description of the button" placement="bottom"></ic-tooltip>`,
+    snippet: `<ic-tooltip target="text-button" label="This is a description of the button" placement="top">
+  <ic-button id="text-button"> Top </ic-button>
```

Comment:
Can we give the buttons unique IDs please. Perhaps add an index number at the end `text-button-0` and remove the white-spaces around the button labels. I don't think they are required.